jormungandr/src/blockchain/process.rs

```diff
@@ -391,7 +389,7 @@ async fn process_and_propagate_new_ref(
     network_msg_box
         .send(NetworkMsg::Propagate(PropagateMsg::Block(header)))
         .await
-        .map_err(|_| "Cannot propagate block to network".into())
+        .map_err(Into::into)
```

Comment:
Nit: I usually see `Ok(...?)` instead of using `Into` for errors, but that's a stylistic choice